packages/reporters/liquidation-reporter/index.ts

```diff
@@ -0,0 +1,17 @@
+const { fetchUmaEcosystemData } = require("./liquidation-reporter");
+const { createExcelSheetFromLiquidationDrawDownData } = require("./excel-writer");
+
+async function fetchDataAndWriteToExcelFile() {
+  const ecosystemData = await fetchUmaEcosystemData();
+  createExcelSheetFromLiquidationDrawDownData(ecosystemData);
+}
+
+fetchDataAndWriteToExcelFile()
+  .then(() => {
+    setTimeout(function() {
```

Comment:
Why is this timeout neccessary?

Reply:
without it the script would sometimes not write correctly. I'm not actually sure why.